terraform/github-wif.tf

```diff
@@ -25,9 +25,9 @@ locals {
 resource "google_iam_workload_identity_pool" "default" {
   project = var.project_id
 
-  workload_identity_pool_id = "github-token-minter"          # 32 characters
-  display_name              = "GitHub Token Minter WIF pool" # 32 characters
-  description               = "GitHub Token Minter OIDC identity pool"
+  workload_identity_pool_id = var.name               # 32 characters
```

Comment:
Are we/should we do any validation on the name variable? e.g. only alphanumeric and hyphens?

Reply:
Yes, the name variable has existing validation here https://github.com/abcxyz/github-token-minter/blob/main/terraform/variables.tf#L25

Reply:
For the `custom_name` input variable of the templates, I think we should add validation for the max length of the string but using the CLI to perform validations

Reply:
That makes sense.